knative-operator/cmd/manager/main.go

```diff
@@ -124,6 +125,16 @@ func main() {
 
 	log.Info("Starting the Cmd.")
 
+	go func() {
+		// This web server is unimportant enough to not bother connecting its lifecycle to
+		// signal handling so the process can just tear it down with it.
+		log.Info("Serving CLI artifacts on :8080")
+		http.Handle("/", http.FileServer(http.Dir("/cli-artifacts")))
```

Comment:
Should we use https?

Reply:
Why would we? The external route is secured appropriately and the traffic is not worth to secure any further, imo. We can doublecheck what the other hosted CLIs do though.